projects/core/src/cart/store/actions/multi-cart.action.ts

```diff
@@ -0,0 +1,149 @@
+import { Action } from '@ngrx/store';
+import { MULTI_CART_FEATURE } from '../multi-cart-state';
+import {
+  EntityLoadAction,
+  EntitySuccessAction,
+  EntityFailAction,
+  EntityResetAction,
+} from '../../../state/utils/entity-loader/entity-loader.action';
+import { EntityRemoveAction } from '../../../state/utils/entity/entity.action';
+import { getCartIdByUserId } from '../../utils/utils';
+import { Cart } from '../../../model/cart.model';
+
+export const RESET_FRESH_CART = '[Multi Cart] Reset Fresh Cart';
+
+export const CREATE_MULTI_CART = '[Multi Cart] Create Cart';
+export const CREATE_MULTI_CART_FAIL = '[Multi Cart] Create Cart Fail';
+export const CREATE_MULTI_CART_SUCCESS = '[Multi Cart] Create Cart Success';
+
+export const LOAD_MULTI_CART = '[Multi Cart] Load Cart';
+export const LOAD_MULTI_CART_FAIL = '[Multi Cart] Load Cart Fail';
+export const LOAD_MULTI_CART_SUCCESS = '[Multi Cart] Load Cart Success';
+
+export const MERGE_MULTI_CART = '[Multi Cart] Merge Cart';
+export const MERGE_MULTI_CART_SUCCESS = '[Multi Cart] Merge Cart Success';
+
+export const RESET_MULTI_CART_DETAILS = '[Multi Cart] Reset Cart Details';
+
+export const CLEAR_MULTI_CART = '[Multi Cart] Clear Cart';
+
+export const SET_FRESH_CART_ID = '[Multi Cart] Set Fresh Cart Id';
+
+export const SET_FAKE_LOADING_CART = '[Multi Cart] Set Fake Loading Cart';
+
+export const REMOVE_CART = '[Multi Cart] Remove Cart';
+
+export class ResetFreshCart extends EntityResetAction {
+  readonly type = RESET_FRESH_CART;
+  constructor() {
+    super(MULTI_CART_FEATURE, 'fresh');
+  }
+}
+
+export class SetFreshCart extends EntitySuccessAction {
+  readonly type = SET_FRESH_CART_ID;
```

Comment:
Could we align the type with the name of action? `SET_FRESH_CART_ID` vs `SetFreshCart`. From the type I deduce it's Cart's data, not the id.